### PR TITLE
P17 AUV ingestion csvs

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: CP05MOAS-A6263/CP05MOAS-A6263_R00013_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.auv_eng.auv.auv_eng_auv_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00013/*.txt,CP05MOAS-A6263-00-ENG000000,recovered_host,Available,
+mi.dataset.driver.flort_kn.auv.flort_kn_auv_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00013/*.txt,CP05MOAS-A6263-01-FLORTN000,recovered_host,Available,
+mi.dataset.driver.dosta_ln.auv.dosta_ln_auv_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00013/*.txt,CP05MOAS-A6263-02-DOSTAN000,recovered_host,Available,
+mi.dataset.driver.ctdav_nbosi.auv.ctdav_nbosi_auv_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00013/*.txt,CP05MOAS-A6263-03-CTDAVN000,recovered_host,Available,
+mi.dataset.driver.nutnr_n.nutnr_n_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00013/DATA/*.SUN,CP05MOAS-A6263-04-NUTNRN000,recovered_host,Available,
+mi.dataset.driver.adcpa_n.adcpa_n_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00013/ADCP/*.ADC,CP05MOAS-A6263-05-ADCPAN000,recovered_host,Available,
+mi.dataset.driver.parad_n.auv.parad_n_auv_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00013/*.txt,CP05MOAS-A6263-06-PARADN000,recovered_host,Available,
```
Comment (snwhite) [inline CP05MOAS-A6263/CP05MOAS-A6263_R00013_ingest.csv:8]: Looks good.

File: CP05MOAS-A6264/CP05MOAS-A6264_R00016_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.auv_eng.auv.auv_eng_auv_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6264/R00016/*.txt,CP05MOAS-A6264-00-ENG000000,recovered_host,Available,
+mi.dataset.driver.flort_kn.auv.flort_kn_auv_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6264/R00016/*.txt,CP05MOAS-A6264-01-FLORTN000,recovered_host,Available,
+mi.dataset.driver.dosta_ln.auv.dosta_ln_auv_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6264/R00016/*.txt,CP05MOAS-A6264-02-DOSTAN000,recovered_host,Available,
+mi.dataset.driver.ctdav_nbosi.auv.ctdav_nbosi_auv_driver,/omc_data/whoi/OMC/CP05MOAS-A6264/R00016/*.txt,CP05MOAS-A6264-03-CTDAVN000,recovered_host,Available,
+mi.dataset.driver.nutnr_n.nutnr_n_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6264/R00016/DATA/*.SUN,CP05MOAS-A6264-04-NUTNRN000,recovered_host,Available,
+mi.dataset.driver.adcpa_n.adcpa_n_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6264/R00016/ADCP/*.ADC,CP05MOAS-A6264-05-ADCPAN000,recovered_host,Available,
+mi.dataset.driver.parad_n.auv.parad_n_auv_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6264/R00016/*.txt,CP05MOAS-A6264-06-PARADN000,recovered_host,Available,
```
Comment (snwhite) [inline CP05MOAS-A6264/CP05MOAS-A6264_R00016_ingest.csv:8]: Looks good.

**Review by snwhite — APPROVED** [review]

File: CP05MOAS-A6263/CP05MOAS-A6263_R00013_ingest.csv (hunk 1 above)
Comment (rtravis-whoi) [reply CP05MOAS-A6263/CP05MOAS-A6263_R00013_ingest.csv:8]: looks good - data available.

File: CP05MOAS-A6264/CP05MOAS-A6264_R00016_ingest.csv (hunk 2 above)
Comment (rtravis-whoi) [reply CP05MOAS-A6264/CP05MOAS-A6264_R00016_ingest.csv:8]: looks good - data available.

**Review by rtravis-whoi — APPROVED** [review]

**Pull request merged**